substrate-archive-backend/src/frontend.rs

```diff
@@ -17,44 +17,43 @@
 mod client;
 mod executor;
 pub use self::client::{Client, GetMetadata, GetRuntimeVersion};
+use self::executor::ArchiveExecutor;
+use futures::{task::SpawnExt, Future};
 use sc_client_api::{
     execution_extensions::{ExecutionExtensions, ExecutionStrategies},
     ExecutionStrategy,
 };
-// use sc_client_db::Backend;
-use self::executor::ArchiveExecutor;
-use crate::{backend::database::ReadOnlyDatabase, error::Error as ArchiveError};
-use futures::{task::SpawnExt, Future};
 use sc_executor::{NativeExecutionDispatch, NativeExecutor, WasmExecutionMethod};
 use sp_api::ConstructRuntimeApi;
 use sp_core::traits::SpawnNamed;
 use sp_runtime::traits::{BlakeTwo256, Block as BlockT};
 use std::sync::Arc;
+use substrate_archive_common::{database::ReadOnlyDB, error::Error as ArchiveError};
```

Comment:
Re-export `database::ReadonlyDB` and `error::Error`? so this can be `use substrate_archive_common::{ReadOnlyDB, Error as ArchiveError}`?